js/templates/folder.html

```diff
@@ -10,7 +10,7 @@
 	<a class="folder {{#if specialRole}} icon-{{specialRole}} svg{{/if}}">
 		{{name}}
 		{{#if unseen}}
```

Comment:
We go in only if unseen. Drafts are always unseen. So you need to do further tests here.